internal/corpus/generator.go

```diff
@@ -20,6 +21,13 @@ import (
 	"github.com/spf13/afero"
 )
 
+const (
+	templateTypeCustom = iota
+	templateTypeGoText
+)
+
+var ErrNotValidTemplate = errors.New("please, pass --template-type as one of `placeholder` or `gotext`")
```

Comment:
Replacing backticks with `'` to avoid escaping and rendering issues in terminals.
```suggestion
var ErrNotValidTemplate = errors.New("please, pass --template-type as one of 'placeholder' or 'gotext'")
```